docs/src/main/sphinx/sql/refresh-materialized-view.rst

```diff
@@ -0,0 +1,35 @@
+=========================
+REFRESH MATERIALIZED VIEW
+=========================
+
+Synopsis
+--------
+
+.. code-block:: text
+
+    REFRESH MATERIALIZED VIEW view_name
+
+Description
+-----------
+
+Initially populate or refresh the data stored in the materialized view
+``view_name``. The materialized view must be defined with
+:doc:`create-materialized-view`. Data is retrieved from the underlying tables
+accessed by the defined query.
+
+The initial population of the materialized view is typically processing
+intensive since it reads the data from the source tables and performs physical
+write operations.
+
+The refresh operation can be less intensive, if the underlying data has not
+changed and the connector has implemented mechanism to be aware of that.
+Connectors can also simple remove the data and reinsert it, or even perform
```

Comment:
Let's remove this sentence. I tried to rewrite it, but I'm not sure it's needed

Reply:
done